File: src/main/java/com/jdc/db/sql/query/OracleQueryBuilder.java
/*
 * Copyright 2009 Jeff Campbell. All rights reserved. Unauthorized reproduction
 * is a violation of applicable law. This material contains certain
 * confidential or proprietary information and trade secrets of Jeff Campbell.
 */
package com.jdc.db.sql.query;

import com.jdc.db.shared.query.OracleUtil;

/**
 *
 * @author jeff
 */
public class OracleQueryBuilder extends SQLQueryBuilder {

    @Override
    public String formatIgnoreCaseLikeClause(String column, String value) {
        return OracleUtil.formatIgnoreCaseLikeClause(column, value);
    }
}

File: dbtools-query/src/commonMain/kotlin/org/dbtools/query/sql/OracleQueryBuilder.kt
/*
 * Copyright 2009 Jeff Campbell. All rights reserved. Unauthorized reproduction
 * is a violation of applicable law. This material contains certain
 * confidential or proprietary information and trade secrets of Jeff Campbell.
 */
package org.dbtools.query.sql

import org.dbtools.query.shared.util.OracleUtil

/**
 *
 * @author jeff
 */
class OracleQueryBuilder : SQLQueryBuilder() {
    override fun formatIgnoreCaseLikeClause(field: String, value: String): String {
        return OracleUtil.formatIgnoreCaseLikeClause(field, value)
    }
}
